deployment/articles-dxp/01-saml/05-configuring-sp-and-idp-connections.markdown

```diff
@@ -0,0 +1,110 @@
+# Changing the Settings for Service Provider and Identity Provider Connections
+
+To change the SAML Service Provider Settings, navigate to the Service Provider
+tab. 
+
+The Service Provider tab includes these options:
+
+**Require Assertion Signature?:** Check this box to require SAML assertions
+to be individually signed in addition to the entire SAML message.
+
+| **Note:** Individual assertions need not be signed as long as the SAML response
+| itself is signed. The SP and IdP should always communicate over `https` to have
+| encryption at the transport level.
+| 
+| If you believe man-in-the-middle attacks are possible, the SAML response can be
+| signed. The only reason to sign the assertions is if the SAML response is not
+| signed. In this case, assertions should not only be signed but also encrypted.
+
+**Clock Skew:** Clock skew is a tolerance in milliseconds used by the Service
+Provider for mitigating time differences between the clocks of the Identity
+Provider and the Service Provider. This usually only matters when assertions
+have been made to expire very quickly.
+
+**LDAP Import Enabled:** Check this box to import user information from the
+configured LDAP connection based on the resolved `NameID`. LDAP connections can be
+configured from Instance Settings.
+
+**Sign Authn Requests:** Check this box to sign the `AuthnRequest` even if the
+Identity Provider metadata indicates that it's not required.
+
+**Sign Metadata:** Check this box to sign the metadata XML file.
+
+**SSL Required:** Check this box to reject SAML messages that are not sent
+over HTTPS. This does not affect how URLs are generated.
+
+## Changing the SAML Identity Provider Connection Settings
+
+To configure @product@'s SAML Identity Provider Settings, navigate to the
+Identity Provider Connection tab of the SAML Admin portlet and click the *Edit*
+action button on the IdP you want to configure. 
+
+**Name:** The name of the Identity Provider with which to connect.
+
+**Entity ID:** The Identity Provider's entity ID. This value must match the
+entity ID declared in the Identity Provider metadata.
+
+**Enabled:** Check the box to enable this IdP. 
+
+**Clock Skew:** Clock skew is a tolerance in milliseconds used by the Service
+Provider for mitigating time differences between the clocks of the Identity
+Provider and the Service Provider. This usually only matters when assertions
+have been made to expire very quickly.
+
+**Force Authn:** Check this box to have the Service Provider ask the Identity
+Provider to re-authenticate the user before verifying the user.
+
+**Metadata:** You can provide a URL to the Identity Provider metadata XML file
+or you can manually upload it. If you provide a URL, the XML file is
+automatically retrieved and periodically polled for updates. You can change the
+update interval in System Settings by modifying the
+`saml.metadata.refresh.interval` property which specifies a number of seconds.
```

Comment:
@sez11a

Change: It's better to refer to the property name as it's displayed on the UI: `Runtime Metadata Refresh Interval ` (`com.liferay.saml.runtime.configuration.SamlConfiguration.getMetadataRefreshInterval()`)

`saml.metadata.refresh.interval` was the name of the property in 6.x prior to 3.1.0

Reply:
Done.